client/src/css/PasswordReset.css

```diff
@@ -0,0 +1,57 @@
+.password-reset-block-1 {
+  background-image: url('/asset/images/login-background.png');
+  background-size: cover;
+  height: 100%;
+  width: 100%;
+  position: fixed;
+}
+
+.reg-button {
+  background-color: #8bd8d0;
+  border: 2px solid #8bd8d0;
+  margin-left: 128px;
+}
+
+.reg-button:hover {
+  background-color: #74cec8;
+}
+
+.top-margin {
+  margin-top: 216px;
+}
+
+.password-reset-text {
+  text-align: center;
+  color: #fff;
+  font-weight: bold;
+  font-size: 36px;
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+.first-row-margin {
+  width: 18em;
+}
+
+.form-text {
+  font-weight: bolder;
+  font-size: 15px;
+}
+
+.password-reset-form {
+  max-width: 1024px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.form-text .ant-form-item-label > label::after {
+  content: '';
+}
+
+.form-text .ant-form-item-label > label::before {
+  content: '';
+}
+
+.form-text .ant-form-item-label > label {
+  color: #f5222d;
```

Comment:
This css is the same as the registration page, except for the .first-row-margin part.